packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -221,11 +235,10 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
 
         const dirPath: string = await vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY) as string;
         const homeExtDir: string = FileSystemUtil.getDirPath(dirPath);
-        const environmentPath: string = path.join(homeExtDir, 'environments', environmentRegistryEntry.name, 'nodes');
+        const environmentNodesPath: string = path.join(homeExtDir, 'environments', environmentRegistryEntry.name, 'nodes');
```

Comment:
you don't need all this you can use your variable you have at the top for the `environmentBaseDir`

Reply:
True, will refactor.